modules/custom/d_content/modules/d_content_init/src/Services/ConfigUpdate.php

```diff
@@ -27,17 +28,27 @@ class ConfigUpdate {
    */
   protected $moduleHandler;
 
+  /**
+   * Logger factory.
+   *
+   * @var \Drupal\Core\Logger\LoggerChannelFactory
+   */
+  protected $loggerFactory;
+
   /**
    * ConfigUpdate constructor.
    *
    * @param \Drupal\Core\Config\ConfigFactory $configFactory
    *   Config factory.
    * @param \Drupal\Core\Extension\ModuleHandler $moduleHandler
    *   Module handler.
+   * @param \Drupal\Core\Logger\LoggerChannelFactory $loggerFactory
+   *   Logger factory.
    */
-  public function __construct(ConfigFactory $configFactory, ModuleHandler $moduleHandler) {
+  public function __construct(ConfigFactory $configFactory, ModuleHandler $moduleHandler, LoggerChannelFactory $loggerFactory) {
```

Comment:
W sumie to bezpieczniej jest dawać interface'y zamiast konkretnych klas :thinking:

Reply:
Jeśli jest dostępny odpowiedni interfejs, to zawsze interfejs, bo nasz kod zależy od abstrakcji a nie konkretnej klasy. Interfejs może istnieć dalej, a konkretna klasa zniknać np. w D9.

Reply:
Celna uwaga :)